allensdk/model/biophysical/runner.py

```diff
@@ -53,19 +55,21 @@ def _init_lock(lock):
     global _lock
     _lock = lock
 
-def run(description, sweeps=None, procs=6):
+def run(args, sweeps=None, procs=6):
     '''Main function for simulating sweeps in a biophysical experiment.
 
     Parameters
     ----------
-    description : Config
-        All information needed to run the experiment.
+    args : dict
```

Comment:
Make sure to describe the expected contents of this dict, or point to a schema that does.

Reply:
a pointer to `load_description` so that people know where to look would do the trick.